src/components/generics/MainMenuContribution.js

```diff
@@ -128,7 +129,52 @@ class MainMenuContribution extends Component {
     _historyPush(modulesManager, history, route);
   }
 
-  appBarMenu = () => {
+  fetchSubmenuConfig(modulesManager, allEntries) {
+    const menuConfig = modulesManager.getConf("fe-core", "menus", []);
+    const isMenuConfigEmpty = !menuConfig || menuConfig.length === 0;
```

Comment:
const isMenuConfigEmpty = !(menuConfig?.length);

Reply:
I applied this fix in https://github.com/openimis/openimis-fe-core_js/pull/262/commits/e024531a2a5270d2ef97ea7fe325dd6510a8fe44 that will be merged into this PR.